Services/Interfaces/web_interface/web_interface.py

```diff
@@ -55,6 +55,10 @@ def __init__(self, config):
         self.websocket_instance = None
         self._init_web_settings()
 
+    async def register_new_exchange_impl(self, exchange_id):
+        if exchange_id not in self.registered_exchanges_ids:
+            await self._register_on_channels(exchange_id)
```

Comment:
`self.registered_exchanges_ids.append(exchange_id)` is missing isn't it ?

Reply:
It's handled in ```register_new_exchange``` the super class

Reply:
Oh ok great :+1: